### DISCUSSION TITLE
Texture disappeared on one particular website

### DISCUSSION THREAD

**Discussion by cpoulinn** [google/model-viewer#4636] [Q&A]
Hi!
I have a 3D model that has been working fine on the same page for months without any changes, but suddenly lost a texture. It is showing as white.The same model works on my GitHub page and my developer’s dev site, but not on the published page. The AR works perfectly.

I have tried renaming the files and the texture, as well as reexporting them to no avail.

Do you have any idea of what could be causing this issue? Thank you!

**Comment by elalish** [comment]
Not really - it is a glTF or a GLB? Packing as a GLB is always recommended to avoid problems like this.

**Reply by cpoulinn** [reply]
It is a glb. What throws me off is that it changed without any updates on my end.

**Reply by elalish** [reply]
My best guess is some kind of server change - if you can show us a repro, I can probably tell you for sure.
